src/resources/Organizations/Organizations.ts

```diff
@@ -0,0 +1,27 @@
+import {New} from '../BaseInterfaces';
+import Resource from '../Resource';
+import {CreateOrganizationOptions, ListOrganizationOptions, OrganizationModel} from './OrganizationsInterfaces';
+
+export default class Organizations extends Resource {
+    static baseUrl = '/rest/organizations/';
+
+    list(options?: ListOrganizationOptions) {
+        return this.api.get<OrganizationModel[]>(this.buildPath(Organizations.baseUrl, options));
+    }
+
+    create(options: CreateOrganizationOptions) {
```

Comment:
This method should accept an org model as first param I think

Reply:
We cannot send a body to the backend (at least not in swagger). We need to create the org and the only parameter required is the name and then update it AFAIK

Reply:
true